packages/block-library/src/cover/edit/index.js

```diff
@@ -199,6 +202,22 @@ function CoverEdit( {
 			averageBackgroundColor
 		);
 
+		if ( backgroundType === IMAGE_BACKGROUND_TYPE && mediaAttributes.id ) {
+			const { imageDefaultSize } = getSettings();
+
+			// Try to use the previous selected image size if its available
+			// otherwise try the default image size or fallback full size.
```

Comment:
```suggestion
			// Try to use the previous selected image size if it's available
			// otherwise try the default image size or fallback to full size.
```
Nit: Typos